src/ashell/ashell.h

```diff
@@ -3,6 +3,11 @@
 #ifndef __ashell_h__
 #define __ashell_h__
 
+// Switches between  console and IDE protocol mode.
+#define ASHELL_IDE_PROTOCOL   1
```

Comment:
I'm sure this was just for testing, but ASHELL_IDE_PROTOCOL should be set in the Makefile based on whether we are building the IDE or CLI version of ashell.  The same goes for ASHELL_IDE_DBG, though it should likely be a build argument to turn it on.